### PR TITLE
Implementing MCH MisAligner

### PR DESCRIPTION
**Pull request by javierecc**
Implementation of MCH MisAligner code based on AliRoot AliMUONGeometryMisAligner.

### CODE REVIEW

File: Detectors/MUON/MCH/Geometry/Transformer/include/MCHGeometryTransformer/Transformations.h [hunk 1]
```diff
@@ -24,13 +24,13 @@ class TGeoManager;
 namespace o2::mch::geo
 {
 
-/** A geometry transformation creator must be able to 
+/** A geometry transformation creator must be able to
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Geometry/Transformer/include/MCHGeometryTransformer/Transformations.h]: the modifications in this file (and in the Transformations.cxx) are just incidental, could you remove them ?

**Comment by aphecetche** [comment]
@javierecc thanks for the PR and sorry for late reply. 

As you've added a new (sub)directory MisAligner, could you please maybe add a README.md (and/or amend the README in the parent Geometry directory) in that directory briefly explaining what it is for (e.g. what part it plays in the alignement big picture) ?

**Comment by javierecc** [comment]
> 
Thanks @aphecetche! README added.

File: Detectors/MUON/MCH/Geometry/Transformer/include/MCHGeometryTransformer/Transformations.h (hunk 1 above)
Comment (javierecc) [reply Detectors/MUON/MCH/Geometry/Transformer/include/MCHGeometryTransformer/Transformations.h]: Yes ... do you have any magic command for that or do I need to make a new commit to revert back those two files?

Comment (aphecetche) [reply Detectors/MUON/MCH/Geometry/Transformer/include/MCHGeometryTransformer/Transformations.h]: I don't do magic :wink:

One option (as you're starting to have anyway several commits that are "fixing" commits, not feature commits), would be to recreate a clean state of commit(s) 

```
git reset dev 
```

=> all your modifications will now be "un"committed and appear as "not staged for commit" in "git status" command result

Discard your local changes on the files you don't want to modify : 

```
git restore Detectors/MUON/MCH/Geometry/Transformer/include/MCHGeometryTransformer/Transformations.h
git restore Detectors/MUON/MCH/Geometry/Transformer/src/Transformations.cxx
```

add back the files you want to commit, e.g. : 

```
git add . (or git add filenames if you want to be more selective)
```

commit 

```
git commit -m 'some message'
```

check that every still compiles and that you've not lost anything at this stage.

Finally, as your branch history has been changed, you'll need to force push : 

```
git push -f javierrc  (assuming your remote is called javierrc on your workdir)
```

**Comment by javierecc** [comment]
Closing pull request to open a new one  ... something went wrong in my branch.

**Pull request closed without merging**